src/tpm2.c

```diff
@@ -5686,6 +5706,20 @@ void TPM2_PrintBin(const byte* buffer, word32 length)
         length -= sz;
     }
 }
+
+void TPM2_PrintAuth(const TPMS_AUTH_COMMAND* authCmd)
+{
+    if (authCmd == NULL)
+        return;
+
+    printf("authCmd:\n");
+    printf("sessionHandle=0x%7X\n", authCmd->sessionHandle);
```

Comment:
Why not `0x%08X` or `0x%8X`?

Reply:
session Handles are 7 digits, other TPM objects are loaded into 8 digit handles

```
wolftpm/tpm2.h:    TPM_HT_PCR              = 0x00,
wolftpm/tpm2.h:    TPM_HT_NV_INDEX         = 0x01,
wolftpm/tpm2.h:    TPM_HT_HMAC_SESSION     = 0x02,
wolftpm/tpm2.h:    TPM_HT_LOADED_SESSION   = 0x02,
wolftpm/tpm2.h:    TPM_HT_POLICY_SESSION   = 0x03,
wolftpm/tpm2.h:    TPM_HT_ACTIVE_SESSION   = 0x03,

notice the leading zero, of course we could print that too to show it is a 7 digit session handle

wolftpm/tpm2.h:    TPM_HT_PERMANENT        = 0x40,
wolftpm/tpm2.h:    TPM_HT_TRANSIENT        = 0x80,
wolftpm/tpm2.h:    TPM_HT_PERSISTENT       = 0x81,

```

I will fix it.